hail/src/main/scala/is/hail/cxx/Definition.scala

```diff
@@ -59,4 +73,16 @@ class FunctionBuilder(prefix: String, args: Array[Variable], returnType: Type) {
 
   def result(): Function = new Function(returnType, prefix, args, statements.result().mkString("\n"))
 
+  def defaultReturn: Code = {
+    if (returnType == "long")
```

Comment:
I find this kind of dicey (what if it is is a double?) but it's good enough for now and let's fix by using an exceptions-based solution.